babel.config.js

```diff
@@ -1,5 +1,10 @@
 module.exports = {
   presets: [
     "@vue/cli-plugin-babel/preset"
-  ]
+  ],
+  "env": {
+    "production": {
+      "plugins": ["transform-remove-console"]
```

Comment:
What is this for?

Reply:
I use this config to remove all consoles in production, actually there is 1 more parameter
```js
// { "exclude": [ "error", "warning"] }

"plugins": [ ["transform-remove-console", { "exclude": [ "error", "warn"] }] ]
```

so it will still display console errors and warnings. Any input, mas @agustinustheo?

Reply:
nope